website/docs/r/workloads_sap_virtual_instance.html.markdown

```diff
@@ -0,0 +1,482 @@
+
+---
+
+A `single_server_configuration` block supports the following:
+
+* `app_resource_group_name` - (Required) The name of the application Resource Group where SAP system resources will be deployed. Changing this forces a new resource to be created.
+
+~> **Note:** While creating SAP Virtual Instance, service would provision the extra SAP system/component in `app_resource_group_name` which aren't defined in tf config. At this time, when `app_resource_group_name` is different with the Resource Group where SAP Virtual Instance exists, we can set `prevent_deletion_if_contains_resources` to `false` to delete all resources defined in tf config and the resources created in `app_resource_group_name` with `tf destroy`. But when `app_resource_group_name` is same with the Resource Group where SAP Virtual Instance exists, some resources like the subnet defined in tf config cannot be deleted with `tf destroy` since the resources defined in tf config are being referenced by the SAP system/component created in `app_resource_group_name`. So it has to manually delete the resources in `app_resource_group_name` first after the SAP Virtual Instance is deleted, and then the resources in tf config can be deleted successfully for this situation.
+
+* `subnet_id` - (Required) The resource ID of the Subnet for the SAP Virtual Instance. Changing this forces a new resource to be created.
+
+* `virtual_machine_configuration` - (Required) A `virtual_machine_configuration` block as defined below. Changing this forces a new resource to be created.
+
+* `database_type` - (Optional) The supported SAP database type. Possible values are `DB2` and `HANA`. Changing this forces a new resource to be created.
+
+* `disk_volume_configuration` - (Optional) One or more `disk_volume_configuration` blocks as defined below. Changing this forces a new resource to be created.
+
+* `secondary_ip_enabled` - (Optional) Is a secondary IP Address that should be added to the Network Interface on all VMs of the SAP system being deployed enabled? Defaults to `false`. Changing this forces a new resource to be created.
+
+* `virtual_machine_full_resource_names` - (Optional) A `virtual_machine_full_resource_names` block as defined below. Changing this forces a new resource to be created.
+
+---
+
+A `disk_volume_configuration` block supports the following:
+
+* `volume_name` - (Required) The name of the DB volume of the disk configuration. Possible values are `backup`, `hana/data`, `hana/log`, `hana/shared`, `os` and `usr/sap`. Changing this forces a new resource to be created.
+
+* `count` - (Required) The total number of disks required for the concerned volume. Changing this forces a new resource to be created.
+
+* `size_in_gb` - (Required) The size of the Disk in GB. Changing this forces a new resource to be created.
+
+* `sku_name` - (Required) The name of the Disk SKU. Changing this forces a new resource to be created.
```

Comment:
can we add details of the possible values here?

Reply:
Updated